<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTCH-5CD105HTG9" name="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters" tests="10" failures="2" timestamp="11 Jun 2021 11:10:27 GMT" time="96.006" errors="0">
  <testcase name="MethodologyItem_FsliFliters_DeleteFinancialStatementFilters_404" time="9.828" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters"/>
  <testcase name="MethodologyItem_FsliFliters_DeleteFinancialStatementFilters_204" time="11.055" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters">
    <failure type="java.lang.AssertionError" message="expected [204] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [204] but found [500]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters.MethodologyItem_FsliFliters_DeleteFinancialStatementFilters_204(Restassured_Automation_MethodologyItemFsliFilters.java:610)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- MethodologyItem_FsliFliters_DeleteFinancialStatementFilters_204 -->
  <testcase name="MethodologyItem_FsliFliters_PatchFinancialStatementFilters_404" time="9.419" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters"/>
  <testcase name="MethodologyItem_FsliFliters_CreateFinancialStatementFilters_409" time="10.343" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters"/>
  <testcase name="MethodologyItem_FsliFliters_PatchFinancialStatementFilters_400" time="9.712" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters"/>
  <testcase name="MethodologyItem_FsliFliters_PatchFinancialStatementFilters_204" time="10.561" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters"/>
  <testcase name="MethodologyItem_FsliFliters_PatchFinancialStatementFilters_409" time="10.238" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters">
    <failure type="java.lang.AssertionError" message="expected [409] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [409] but found [500]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters.MethodologyItem_FsliFliters_PatchFinancialStatementFilters_409(Restassured_Automation_MethodologyItemFsliFilters.java:409)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- MethodologyItem_FsliFliters_PatchFinancialStatementFilters_409 -->
  <testcase name="MethodologyItem_FsliFliters_CreateFinancialStatementFilters_200" time="8.897" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters"/>
  <testcase name="MethodologyItem_FsliFliters_CreateFinancialStatementFilters_404" time="7.528" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters"/>
  <testcase name="MethodologyItem_FsliFliters_CreateFinancialStatementFilters_400" time="8.425" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters"/>
</testsuite> <!-- restassured.automation.testcases.Restassured_Automation_MethodologyItemFsliFilters -->
